src/components/card/card.scss

```diff
@@ -112,9 +112,16 @@ $_focus-overflow: (size.$edge-large * -1);
     grid-template-columns: repeat(3, 1fr); /* 2 */
     grid-template-rows: 1fr repeat(3, minmax(0, auto)) 1fr; /* 3 */
 
+    .c-card__cover {
+      block-size: 100%; /* 4 */
+      inline-size: 100%; /* 4 */
+    }
+
     /**
      * When the cover image is circular (and thus taller), give it a smaller
      * amount of space and keep the text left-aligned.
+     *
+     * 1. Fixes bug where the circle cover wasn't a circle at smallest viewports
```

Comment:
Nit: Could we make this comment easier to follow by moving it closer to the block it pertains to?

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/2057/commits/12912dbd0fc35c8dfefab39d2bc83e063b19331c